README.md

```diff
@@ -1 +1,826 @@
-# message-tracing-with-stream-processor
\ No newline at end of file
+[![Build Status](https://travis-ci.org/ballerina-guides/message-tracing-with-stream-processor.svg?branch=master)](https://travis-ci.org/ballerina-guides/message-tracing-with-stream-processor)
+
+# Integration with WSO2 Stream Processor
+
+WSO2 stream processor provides us with distributed message tracing. The Distributed Message Tracer allows you to trace the events which are produced while ballerina services serve for requests. The ballerina services send these tracing data as WSO2 events.
+        
+  >This guide provides instructions on how Ballerina can be used to integrate with Stream-Processor Message Tracer.
+  
+The following are the sections available in this guide.
```

Comment:
Any reason why we don't have a section for Deployment and Observability? Normally we have those.

Reply:
The ultimate goal of this guide is to observe Ballerina services through SP Message Tracer. Therefore this guide is based on observability.  This is the reason to ignore the Observability section.
The testing part covers the Deployment section. Therefore Deployment section is not included.